### PR TITLE
[docs] build java with ssl

### PR DESCRIPTION
**Pull request by ddhodge**
Building a Java app with SSL #8436
https://deploy-preview-8910--infallible-bardeen-164bc9.netlify.app/latest/quick-start/build-apps/java/ysql-jdbc-ssl/

### CODE REVIEW

**Review by nchandrappa — APPROVED** [review]
LGTM.

File: docs/content/latest/quick-start/build-apps/java/ysql-jdbc-ssl.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+</ul>
+
+## Prerequisites
+
+This tutorial assumes that:
+
+- YugabyteDB is up and running. If you are new to YugabyteDB, you can download, install, and have YugabyteDB up and running within five minutes by following the steps in [Quick start](../../../../quick-start/).
+- Java Development Kit (JDK) 1.8, or later, is installed. JDK installers for Linux and macOS can be downloaded from [OpenJDK](http://jdk.java.net/), [AdoptOpenJDK](https://adoptopenjdk.net/), or [Azul Systems](https://www.azul.com/downloads/zulu-community/).
+- [Apache Maven](https://maven.apache.org/index.html) 3.3 or later, is installed.
+- [OpenSSL](https://www.openssl.org/) is installed.
```
Comment (polarweasel) [inline docs/content/latest/quick-start/build-apps/java/ysql-jdbc-ssl.md:57]: What versions do we support? 1.1.1 at minimum?

File: docs/content/latest/quick-start/build-apps/java/ysql-jdbc-ssl.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+## Set up SSL certificates for Java applications
+
+To build a Java application that connects to YugabyteDB over an SSL connection, you need the root certificate (`ca.crt`), and node certificate (`yugabytedb.crt`) and key (`yugabytedb.key`) files. If you have not generated these files, follow the instructions in [Create server certificates](../../../../secure/tls-encryption/server-certificates/).
+
+1. Download the certificate (`yugabytedb.crt`, `yugabytedb.key`, and `ca.crt`) files (see [Copy configuration files to the nodes](../../../../secure/tls-encryption/server-certificates/#copy-configuration-files-to-the-nodes)).
+
+1. If you do not have access to the system `cacerts` Java truststore you can create your own truststore.
```
Comment (polarweasel) [inline docs/content/latest/quick-start/build-apps/java/ysql-jdbc-ssl.md:65]: Double check, but isn't trust store two words, generally?

File: docs/content/latest/quick-start/build-apps/java/ysql-jdbc-ssl.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+    ```sh
+    $ keytool -keystore ybtruststore -alias ybtruststore -import -file ca.crt
+    ```
+
+1. Verify the `yugabytedb.crt` client certificate with `ybtruststore` by running the following command:
```
Comment (polarweasel) [inline docs/content/latest/quick-start/build-apps/java/ysql-jdbc-ssl.md:71]: You could probably eliminate all the "by running the following command" instances

**Review by polarweasel — CHANGES_REQUESTED** [review]
Looking good. Couple minor comments, then merge away

File: docs/content/latest/quick-start/build-apps/java/ysql-jdbc-ssl.md (hunk 2 above)
Comment (ddhodge) [reply docs/content/latest/quick-start/build-apps/java/ysql-jdbc-ssl.md:65]: no, the Java truststore appears to be 1 word, but the capitalization seems to change with the wind, even in Oracle's docs

**Pull request merged**